proto/private/BUILD.protoc

```diff
@@ -0,0 +1,8 @@
+load("@rules_proto//proto/private:dependencies.bzl", "protoc_binary")
+
+protoc_binary(
```

Comment:
I'm using a genrule now for protoc_binary to try to avoid depending on bash.
However genrule outputs are non-configurable, so select doesn't seem to work.

Reply:
dome